pkg/cvo/updatepayload.go

```diff
@@ -315,3 +318,25 @@ func findUpdateFromConfigVersion(config *configv1.ClusterVersion, version string
 	}
 	return configv1.Update{}, false
 }
+
+func (optr *Operator) defaultClusterProfileRetriever() ClusterProfileRetriever {
+	return &clusterProfileRetriever{
+		cmConfigLister: optr.cmConfigLister,
+	}
+}
+
+type clusterProfileRetriever struct {
+	cmConfigLister listerscorev1.ConfigMapNamespaceLister
+}
+
+func (r *clusterProfileRetriever) RetrieveClusterProfile(_ context.Context) (string, error) {
+	configMap, err := r.cmConfigLister.Get(internal.ClusterProfileConfigMap)
+	if err != nil {
+		if apierrors.IsNotFound(err) {
+			klog.Infof("Cannot get ConfigMap openshift-config/cluster-profile, using default profile: %v", err)
```

Comment:
Instead of hard-coding `openshift-config/cluster-profile` here, can we use `%s/%s`, `ConfigNamespace`, and `ClusterProfileConfigMap`?

Reply:
done :+1: